.github/workflows/scanner-nvd-update.yaml

```
@@ -36,7 +36,14 @@ jobs:
         set -eu
         dir=$(mktemp -d)
         python3 .github/workflows/scripts/scanner-update-nvd-feeds.py "$dir"
-        zip -j nvd-feeds.zip "$dir"/*.nvd.json
+        if [ "$(find "$dir" -name "*.nvd.json" | wc -l)" -gt 0 ]; then
+          if jq . -r "$dir"/*.nvd.json > /dev/null 2>&1; then
```

This will fail if there are no `*.json`, so the weird `find` above is redundant.

```
dir=foobar
jq . -r "$dir"/*.nvd.json                                                                                                                                                                                                             
no matches found: foobar/*.nvd.json
```

Also the `if jq`  is also unnecessary, since bash will fail if jsons are invalid in the `jq` call (also using `2>&1` would hide stderr which could be useful to determine why the files are bad).

I think it's best if we iterate over each file, so we can pinpoint the file that is broken:

```suggestion
          echo "Listing JSON files"
          ls -l "$dir"/*.nvd.json # if no *.json this will fail, and print $dir
          for f in "$dir"/*.nvd.json; do
               echo "Checking contents of '$f' with 'jq' and adding to zip"
               jq . $f >/dev/null
               zip -j nvd-feeds.zip $f
           done
```

---

Saying all that, this is a little bit beyond the scope of the PR description, so maybe we should leave this behind and fix after the time off.